cloudinit/config/cc_mounts.py

```diff
@@ -42,32 +59,35 @@
 with ``filename``, the size of the swap file with ``size`` maximum size of
 the swap file if using an ``size: auto`` with ``maxsize``. By default no
 swap file is created.
+"""
```

Comment:
We should update the blurb mentioning the default:
```
It defaults to::
    mount_default_fields: [none, none, "auto", "defaults,nobootwait", "0", "2"]
On a systemd booted system that default is the mostly equivalent::
    mount_default_fields: [none, none, "auto",
       "defaults,nofail,x-systemd.requires=cloud-init.service", "0", "2"]
Note that `nobootwait` is an upstart specific boot option that somewhat
equates to the more standard `nofail`.
```
Is upstart default anywhere anymore? I think we should just specify the systemd default as the only default and then mention that other init systems may vary in their defaults.